src/ucp/proto/proto_multi.c

```diff
@@ -150,12 +150,15 @@ ucs_status_t ucp_proto_multi_init(const ucp_proto_multi_init_params_t *params,
            Due to floating-point accuracy, max_frag may be too large. */
         max_frag = ucs_double_to_sizet(lane_perf->bandwidth / max_frag_ratio,
                                        lane_perf->max_frag);
+        if (mpriv->num_lanes == 1) {
```

Comment:
can you pls add comment?

Reply:
this would cause sending max_frag on first lane every time it's used, and not only for the first fragment in the message, and it's not a desired behavior